# file: /usr/lib/python3/dist-packages/urllib3/exceptions.py
# hypothesis_version: 6.135.24

['%s: %s', 'Failed to parse: %s', 'Unknown', 'localhost']